demo/kitchen_sink/run.py

```diff
@@ -152,6 +152,7 @@ def fn(
     * 3,
     theme="default",
     title="Kitchen Sink",
+    cache_examples=False,
```

Comment:
New bug: Can't cache dataframes in examples lol. Might be fixed by #1967 

https://huggingface.co/spaces/gradio-test-deploys/3.1.4_kitchen_sink

Reply:
I'll check and fix if needed in #1967